
<launch>
    <remap from="trigger_time" to="controllers/imu_filter_controller/gimbal_imu/trigger_time"/>

    <arg name="enable_imu_trigger" default="false"/>
    <arg name="camera_name" default="galaxy_camera"/>
    <arg name="camera_frame" default="camera_optical_frame"/>

    <node pkg="nodelet" type="nodelet" name="vision_nodelet" args="manager"
          output="screen"/>
    <node pkg="nodelet" type="nodelet" name="galaxy_camera"
          args="load galaxy_camera/GalaxyCameraNodelet vision_nodelet"
          output="screen">
        <param name="image_width" value="1280"/>
        <param name="image_height" value="1024"/>
        <param name="pixel_format" value="bgr8"/>
        <param name="camera_info_url" value=""/>
        <param name="camera_frame_id" value="$(arg camera_frame)"/>
        <param name="camera_sn" value="KE0200060392"/>
        <param name="enable_imu_trigger" value="$(arg enable_imu_trigger)"/>
        <param name="raising_filter_value" value="0"/>
        <param name="exposure_value" value="2000."/>
        <param name="exposure_auto" value="false"/>
    </node>

</launch>
